assets/app/scripts/services/data.js

```diff
@@ -236,16 +236,22 @@ angular.module('openshiftConsole')
 
   // objects:   Array of API object data(eg. [{ kind: "Build", parameters: { ... } }] )
   // context:   API context (e.g. {project: "..."})
-  // opts:      http - options to pass to the inner $http call
+  // opts:      action - defines the REST action that will be called
+  //                   - available actions: Create, Update
+  //                   - default action: Create
```

Comment:
Your comment says "Create" and "Update", but you check for "update" below. (I think I prefer the lowercase strings.)